Tools-Override/tests.targets

```diff
@@ -246,10 +246,8 @@
   </Target>
 
   <Target Name="CheckTestPlatforms">
-    <GetTargetMachineInfo Condition="'$(TargetOS)' == ''">
-      <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
-    </GetTargetMachineInfo>
     <PropertyGroup>
+      <TargetOS Condition="'$(TargetOS)' == ''">$(DefaultOSGroup)</TargetOS>
```

Comment:
OK can you clean those out then if they are dead. I would like to keep these things consistent